CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Unreleased 
+- Add ignoreTransactions and ignoreErrors to options ([#1391](https://github.com/getsentry/sentry-dart/issues/1391)) ([#2207](https://github.com/getsentry/sentry-dart/pull/2207))
```

Comment:
```suggestion

### Features 
- Add `ignoreTransactions` and `ignoreErrors` to options ([#2207](https://github.com/getsentry/sentry-dart/pull/2207))
```

linking the PR should be good enough. 

also I like to add a little code example below to show users how they could utilize these options.

example: https://github.com/getsentry/sentry-dart/blob/main/CHANGELOG.md#features-7